dcm4che-net/src/main/java/org/dcm4che3/net/Device.java

```diff
@@ -1084,7 +1063,11 @@ public ApplicationEntity getApplicationEntity(String aet) {
      * @return AE titles of this device, including alias AE titles
      */
     public Collection<String> getApplicationAETitles() {
-        return aliasApplicationEntitiesMap.keySet();
+        HashSet<String> aeTitles = new HashSet<>();
```

Comment:
Nit-pick; usually on the left side you declare it as "Set" interface, rather than concrete class, but not a big deal.